ui/decredmaterial/progressbar.go

```diff
@@ -152,7 +152,10 @@ func (mp *MultiLayerProgressBar) progressBarLayout(gtx C) D {
 	}
 
 	calProgressWidth := func(progress int) int {
-		val := (progress / mp.total) * 100
+		val := 1
+		if mp.total != 0 {
+			val = (progress / mp.total) * 100
+		}
```

Comment:
Why are we making this update?

Reply:
in my case for some reason mp.total = 0 causes the app to crash when switching to staking tab, it should be checked before doing the division

Reply:
i think converting to float should fix the crash